src/providers/pkcs11_provider/utils.rs

```diff
@@ -48,11 +48,13 @@ pub fn rv_to_response_status(rv: CK_RV) -> ResponseStatus {
         s @ CKR_CURVE_NOT_SUPPORTED
         | s @ CKR_DOMAIN_PARAMS_INVALID
         | s @ CKR_FUNCTION_NOT_SUPPORTED => {
-            error!("Not supported value ({:?})", s);
+            if crate::utils::GlobalConfig::log_error_details() {
+                error!("Not supported value ({:?})", s);
```

Comment:
Is it not a `format_error` here?

Reply:
Notice the missing `else` branch - I thought that printing `Not supported value` while returning `PsaErrorNotSupported` is kinda redundant... In the other ones at least we're giving some more info

Reply:
(oh, also, it's not a `format_error` because `s` doesn't implement `Display` 😬 which is required for `format_error` (because it uses `{}`, not `{:?}`)